Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/TetrahedronFEMForceField.inl

```diff
@@ -1349,6 +1349,12 @@ void TetrahedronFEMForceField<DataTypes>::init()
         if (youngModulus[i]>maxYoung) maxYoung=youngModulus[i];
     }
 
+    const Real& poissonRatio = _poissonRatio.getValue();
+    if (poissonRatio<0 || poissonRatio>=0.5)
+    {
+        msg_error() << "FEM Poisson Ratio in Hooke's law should be in [0,0.5[.";
```

Comment:
FYI I've just copied the data's info. I don't think we should use a negative ratio with Hooke's law. https://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.661.2128&rep=rep1&type=pdf

```cpp
Data<Real> _poissonRatio; ///< FEM Poisson Ratio [0,0.5[
```
```cpp
_poissonRatio(initData(&_poissonRatio,(Real)0.45f,"poissonRatio","FEM Poisson Ratio in Hooke's law [0,0.5["))
```

To be honest my first intention was to set the component to an invalid state but I got discouraged by the 45 methods to identify/update. In the current implementation adding a `return;` after the error message (which would set the state of the component to invalid) will just make the simulation crash, since it is not used at all elsewhere. 
I don't think that clamping the value is a good idea either, same as for setting the default value.

Reply:
FYI we need at least a message because of this -> https://github.com/SofaDefrost/SoftRobots/discussions/130 (nan values if you set the ratio to 0.5).